applications/plugins/Communication/examples/VRPN/primitives/Text/Receiver/receiver.scn

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+<Node name="root">
+
+    <LightManager />
+    <SpotLight name="light1" color="1 1 1" position="0 80 25" direction="0 -1 0" cutoff="100" exponent="1" />
+
+    <Node name="Visu" tags="Visual">
+        <OglModel  name="VisualModel" fileMesh="mesh/sphere_01.obj" />
+    </Node>
+
+    <RequiredPlugin name="Communication" pluginName="Communication" />
+    <ServerCommunicationVRPN name="vrpn1" job="receiver" address="localhost"/>
+    <CommunicationSubscriber name="sub1" communication="@vrpn1" subject="Mouse0" target="@light1" datas="aNewStringValue"/>
```

Comment:
Mouse0 is ambiguous. We should use "Testing" instead.

Reply:
Okay yes, we can do that. Should I add that change in the next commit?